src/loaders/lottie/tvgLottieBuilder.cpp

```diff
@@ -1129,12 +1129,14 @@ void LottieBuilder::updateMaskings(LottieLayer* layer, float frameNo)
     auto pMask = static_cast<LottieMask*>(layer->masks[0]);
     auto pMethod = pMask->method;
     auto opacity = pMask->opacity(frameNo);
+    //TODO: Once path direction support is implemented, ensure that the direction is ignored here
+    auto offset = LottieOffsetModifier(pMask->expand(frameNo));
 
     auto pShape = layer->pooling();
     pShape->reset();
     pShape->fill(255, 255, 255, opacity);
     pShape->transform(layer->cache.matrix);
-    pMask->pathset(frameNo, P(pShape)->rs.path.cmds, P(pShape)->rs.path.pts, nullptr, nullptr, nullptr, exps);
+    pMask->pathset(frameNo, P(pShape)->rs.path.cmds, P(pShape)->rs.path.pts, nullptr, nullptr, &offset, exps);
```

Comment:
this brings the unnecessary path modification. let's skip the offest pass if it's invalid value.